integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/tests/ServletApiTestApp.java

```diff
@@ -93,6 +97,63 @@ private void testHttpUrlConnection(AbstractServletContainerIntegrationTest test)
         assertThat(span.get("context").get("http").get("status_code").intValue()).isEqualTo(200);
     }
 
+    private void testTransactionReportingWithForward(AbstractServletContainerIntegrationTest test) throws Exception {
+        String pathToTest = "/simple-webapp" + "/forward";
+        boolean isExistForwardSpan = false;
+        boolean isExistDbH2QuerySpan = false;
+        List<String> spanTypes = Arrays.asList("db.h2.query", "servlet.request-dispatcher.forward");
+        List<String> expectedValues = Arrays.asList("FORWARD", "FORWARD /echo");
```

Comment:
ah, so it works on some application server but on others, it doesn't?

Reply:
no, it doesn't work on all application servers.
For this reason I added 2 expected values, but "FORWARD" - is actual.

Reply:
Your ForwardingServlet forwards to `/servlet` but you expect `/echo`. Don't allow two options unless you must. See if there is a way to look for a definitive name, otherwise we are not testing ourselves properly.

Reply:
my mistake, I deleted 2nd element.